pyomo/contrib/doe/tests/test_doe_solve.py

```diff
@@ -11,6 +11,7 @@
 import json
 import logging
 import os.path
+import idaes.core.solvers.get_solver
```

Comment:
What is this import being used for? We generally want to avoid importing anything from IDAES in Pyomo since it creates a circular dependency.

Reply:
@smondal13 In our group manual, I added instructions on making the "good Ipopt" executable accessible for all condo environments in macOS. I also posted this in our group Slack a few weeks ago. Going this will mean you do not need IDAES for the ipopt solver.